scripts/main.js

```diff
@@ -0,0 +1,155 @@
+function menuMobile() {
+  const mobileDisplay = document.getElementById('mobileDisplay');
+  const mobileMenu = document.getElementById('mobileMenu');
+
+  mobileDisplay.classList.toggle('dispNone1');
+  mobileMenu.classList.toggle('dispNone1');
+}
+
+const Speakers = [{
```

Comment:
Great design @fdezcaminero 🤝

- [ ] Kindly fit the text section for each speaker as well as center the whole section properly. The reason is to make your design look like the design given as much as possible.
- Your design
![your speakers](https://user-images.githubusercontent.com/63917725/235015248-6ce277e1-2401-482b-b1ca-bb8d4a896d4f.PNG)
- Given design
![given speakers](https://user-images.githubusercontent.com/63917725/235015278-ac4bfbe3-7890-4d64-9078-05d63f539c91.PNG)